secretmanager/snippets/regional_samples/get_regional_secret_version.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for getting metadata about a regional
+secret version, but not the secret payload.
+"""
+
+import argparse
+
+from google.cloud import secretmanager_v1
+
+
+# [START secretmanager_v1_get_regional_secret_version]
+def get_regional_secret_version(
+    project_id: str, location_id: str, secret_id: str, version_id: str
+) -> secretmanager_v1.GetSecretVersionRequest:
+    """
+    Get information about the given secret version. It does not include the
+    payload data.
+    """
```

Comment:
suggestion: can you provide a docstring explaining the arguments of this function?

Reply:
For the sake of consistency, and limited time, i will be adding the explanation of arguments in a different PR. As of now, they are in consistent with the current samples that are already live.